includes/class-wc-geolocation.php

```diff
@@ -237,7 +238,8 @@ public static function geolocate_ip( $ip_address = '', $fallback = false, $api_f
 	 * @return string
 	 */
 	public static function get_local_database_path( $deprecated = '2' ) {
-		return apply_filters( 'woocommerce_geolocation_local_database_path', WP_CONTENT_DIR . '/uploads/GeoLite2-Country.mmdb', $deprecated );
+		require_once WC_ABSPATH . 'includes/integrations/maxmind-geolocation/class-wc-integration-maxmind-geolocation-database.php';
```

Comment:
This doesn't sound correct, since `WC_Autoloader` should load this integration before `includes/class-wc-geolocation.php` is included.

Reply:
Seems like `class-wc-integrations.php` is included after `class-wc-geolocation.php`, this doesn't sound correct, integrations suppose to work everywhere.